modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,129 @@
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `musthave` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
+
+To remove the content from the managed cluster, set the `complianceType` field to `mustonlyhave`.
+In this mode, the policy ensures that the CR on the cluster is an exact match of what is specified in the policy.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have deployed a managed cluster from a hub cluster running {rh-rhacm}.
+
+* You have installed {cgu-operator-full} on the hub cluster.
+
+.Procedure
+
+. Remove the content that you no longer need from the affected CRs. In this example, the `disableDrain: false` line should be removed from the `SriovOperatorConfig` CR.
++
+.Example CR
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  configDaemonNodeSelector:
+    "node-role.kubernetes.io/$mcp": ""
+  disableDrain: true <1>
+  enableInjector: true
+  enableOperatorWebhook: true
+----
+<1> Remove the `disableDrain: true` line.
+
+. Change the `complianceType` of the affected policies to `mustonlyhave` in the `group-du-sno-ranGen.yaml` file.
++
+.Example YAML
+[source,yaml]
+----
+- fileName: SriovOperatorConfig.yaml
+  policyName: "config-policy"
+  complianceType: mustonlyhave
+----
+
+. Create a `ClusterGroupUpdates` CR and specify the clusters you want to deploy the changes to:
++
+.Example ClusterGroupUpdates CR
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-remove
+  namespace: default
+spec:
+  managedPolicies:
+    - ztp-group.group-du-sno-config-policy
+  enable: false
+  clusters:
+  - spoke1
+  - spoke2
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
+  batchTimeoutAction:
+----
+
+. Create the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f cgu-remove.yaml
+----
+
+. When you are ready to apply the changes, for example, during an appropriate maintenance window, change the value of the `spec.enable` field to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace=default patch clustergroupupgrade.ran.openshift.io/cgu-remove \
+--patch '{"spec":{"enable":true}}' --type=merge
+----
+
+. Check the status of the policies by running the following command:
++
+[source,terminal]
+----
+$ oc get <changed_cr>
```

Comment:
Callout needed to inform a docs user that this is a replaceable field?

Reply:
Maybe explain exactly what is needed here? 

```
. Check the status of the policies by running the following command for the changed CR:
+
[source,terminal]
----
$ oc get <kind> <name>
----
```